extensions/azurePublish/src/services/webAppAzureFunctionService.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { WebSiteManagementClient } from '@azure/arm-appservice';
+import { TokenCredentials } from '@azure/ms-rest-js';
+
+import { AzureFunctionsConfig } from '../azureResourceManager/azureResourceManagerConfig';
+
+import { throwNotImplementedError } from './throwNotImplementedError';
+
+const createWebAppAzureFunctionService = (token: string, subscriptionId: string) => {
```

Comment:
Nit:  I would call this createAzureAppService for now and later call it createAzureWebOrFunctionAppService if I need to distinguish it from something else.